src/styles/card.css

```diff
@@ -0,0 +1,117 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+}
+
+.card__text {
+  padding: 40px 16px 4px;
```

Comment:
Use better margin:top (if this element will have border everything will be broken)

Reply:
agree, let's fix this